app/types/transport/responses/task/create-task.ts

```diff
@@ -10,6 +10,11 @@ export interface CreateTaskResponsePayload {
    * Response sample
    */
   task: Task;
+
+  /**
+   * Updated status object in case task was created with statusId specified
+   */
+  status?: Status | null;
```

Comment:
why it is not inside the `task`?

Reply:
it's not part of task, it's a full status object which is used to update status data on ui